### ISSUE TITLE
Suddenly can't build GLEW anymore

### ISSUE THREAD

**Issue by vanfanel** [nigels-com/glew#399]
Hi there,

So far, I have been building GLEW nicely using:
`make extensions && make SYSTEM=linux-egl-glvnd -j4 install`

...but with current GIT code, I get these:

```
root@raspberrypi:~/src/glew# make SYSTEM=linux-egl-glvnd -j4 install
install -d -m 0755 "/usr/local/include/GL"
sed \
        -e "s|@prefix@|/usr/local|g" \
        -e "s|@libdir@|/usr/local/lib|g" \
        -e "s|@exec_prefix@|/usr/local/bin|g" \
        -e "s|@includedir@|/usr/local/include/GL|g" \
        -e "s|@version@|2.2.0|g" \
        -e "s|@cflags@||g" \
        -e "s|@libname@|GLEW|g" \
        -e "s|@libgl@|-lEGL -lOpenGL|g" \
        -e "s|@requireslib@||g" \
        < glew.pc.in > glew.pc
cc -DGLEW_NO_GLU -DGLEW_BUILD -O2 -Wall -W -Wshadow -pedantic -Iinclude -fPIC -Wcast-qual -ansi -pedantic -fno-stack-protector -DGLEW_EGL  -o tmp/linux-egl-glvnd/default/shared/glew.o -c src/glew.c
cc -DGLEW_NO_GLU -DGLEW_STATIC -O2 -Wall -W -Wshadow -pedantic -Iinclude -fPIC -Wcast-qual -ansi -pedantic -fno-stack-protector -DGLEW_EGL  -o tmp/linux-egl-glvnd/default/static/glew.o -c src/glew.c
install -m 0644 include/GL/wglew.h "/usr/local/include/GL/"
install -d -m 0755 "/usr/local/lib/pkgconfig"
install -m 0644 include/GL/glew.h "/usr/local/include/GL/"
install -d -m 0755 "/usr/local/lib/pkgconfig"
install -m 0644 glew.pc "/usr/local/lib/pkgconfig/"
install -m 0644 include/GL/glxew.h "/usr/local/include/GL/"
install -m 0644 include/GL/eglew.h "/usr/local/include/GL/"
In file included from src/glew.c:34:
include/GL/glew.h:14893:28: warning: redefinition of typedef ‘PFNGLBINDSAMPLERPROC’ [-Wpedantic]
14893 | typedef void (GLAPIENTRY * PFNGLBINDSAMPLERPROC) (GLuint unit, GLuint sampler);
      |                            ^~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:6244:28: note: previous declaration of ‘PFNGLBINDSAMPLERPROC’ with type ‘PFNGLBINDSAMPLERPROC’ {aka ‘void (*)(unsigned int,  unsigned int)’}
 6244 | typedef void (GLAPIENTRY * PFNGLBINDSAMPLERPROC) (GLuint unit, GLuint sampler);
      |                            ^~~~~~~~~~~~~~~~~~~~
In file included from src/glew.c:34:
include/GL/glew.h:14893:28: warning: redefinition of typedef ‘PFNGLBINDSAMPLERPROC’ [-Wpedantic]
14893 | typedef void (GLAPIENTRY * PFNGLBINDSAMPLERPROC) (GLuint unit, GLuint sampler);
      |                            ^~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:6244:28: note: previous declaration of ‘PFNGLBINDSAMPLERPROC’ with type ‘PFNGLBINDSAMPLERPROC’ {aka ‘void (*)(unsigned int,  unsigned int)’}
 6244 | typedef void (GLAPIENTRY * PFNGLBINDSAMPLERPROC) (GLuint unit, GLuint sampler);
      |                            ^~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:14894:28: warning: redefinition of typedef ‘PFNGLDELETESAMPLERSPROC’ [-Wpedantic]
14894 | typedef void (GLAPIENTRY * PFNGLDELETESAMPLERSPROC) (GLsizei count, const GLuint* samplers);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:6245:28: note: previous declaration of ‘PFNGLDELETESAMPLERSPROC’ with type ‘PFNGLDELETESAMPLERSPROC’ {aka ‘void (*)(int,  const unsigned int *)’}
 6245 | typedef void (GLAPIENTRY * PFNGLDELETESAMPLERSPROC) (GLsizei count, const GLuint * samplers);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:14894:28: warning: redefinition of typedef ‘PFNGLDELETESAMPLERSPROC’ [-Wpedantic]
14894 | typedef void (GLAPIENTRY * PFNGLDELETESAMPLERSPROC) (GLsizei count, const GLuint* samplers);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:14895:28: warning: redefinition of typedef ‘PFNGLGENSAMPLERSPROC’ [-Wpedantic]
14895 | typedef void (GLAPIENTRY * PFNGLGENSAMPLERSPROC) (GLsizei count, GLuint* samplers);
      |                            ^~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:6246:28: note: previous declaration of ‘PFNGLGENSAMPLERSPROC’ with type ‘PFNGLGENSAMPLERSPROC’ {aka ‘void (*)(int,  unsigned int *)’}
 6246 | typedef void (GLAPIENTRY * PFNGLGENSAMPLERSPROC) (GLsizei count, GLuint* samplers);
      |                            ^~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:6245:28: note: previous declaration of ‘PFNGLDELETESAMPLERSPROC’ with type ‘PFNGLDELETESAMPLERSPROC’ {aka ‘void (*)(int,  const unsigned int *)’}
 6245 | typedef void (GLAPIENTRY * PFNGLDELETESAMPLERSPROC) (GLsizei count, const GLuint * samplers);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:14896:28: warning: redefinition of typedef ‘PFNGLGETSAMPLERPARAMETERFVPROC’ [-Wpedantic]
14896 | typedef void (GLAPIENTRY * PFNGLGETSAMPLERPARAMETERFVPROC) (GLuint sampler, GLenum pname, GLfloat* params);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:6249:28: note: previous declaration of ‘PFNGLGETSAMPLERPARAMETERFVPROC’ with type ‘PFNGLGETSAMPLERPARAMETERFVPROC’ {aka ‘void (*)(unsigned int,  unsigned int,  float *)’}
 6249 | typedef void (GLAPIENTRY * PFNGLGETSAMPLERPARAMETERFVPROC) (GLuint sampler, GLenum pname, GLfloat* params);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:14895:28: warning: redefinition of typedef ‘PFNGLGENSAMPLERSPROC’ [-Wpedantic]
14895 | typedef void (GLAPIENTRY * PFNGLGENSAMPLERSPROC) (GLsizei count, GLuint* samplers);
      |                            ^~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:6246:28: note: previous declaration of ‘PFNGLGENSAMPLERSPROC’ with type ‘PFNGLGENSAMPLERSPROC’ {aka ‘void (*)(int,  unsigned int *)’}
 6246 | typedef void (GLAPIENTRY * PFNGLGENSAMPLERSPROC) (GLsizei count, GLuint* samplers);
      |                            ^~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:14897:28: warning: redefinition of typedef ‘PFNGLGETSAMPLERPARAMETERIVPROC’ [-Wpedantic]
14897 | typedef void (GLAPIENTRY * PFNGLGETSAMPLERPARAMETERIVPROC) (GLuint sampler, GLenum pname, GLint* params);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:6250:28: note: previous declaration of ‘PFNGLGETSAMPLERPARAMETERIVPROC’ with type ‘PFNGLGETSAMPLERPARAMETERIVPROC’ {aka ‘void (*)(unsigned int,  unsigned int,  int *)’}
 6250 | typedef void (GLAPIENTRY * PFNGLGETSAMPLERPARAMETERIVPROC) (GLuint sampler, GLenum pname, GLint* params);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:14896:28: warning: redefinition of typedef ‘PFNGLGETSAMPLERPARAMETERFVPROC’ [-Wpedantic]
14896 | typedef void (GLAPIENTRY * PFNGLGETSAMPLERPARAMETERFVPROC) (GLuint sampler, GLenum pname, GLfloat* params);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:14898:33: warning: redefinition of typedef ‘PFNGLISSAMPLERPROC’ [-Wpedantic]
14898 | typedef GLboolean (GLAPIENTRY * PFNGLISSAMPLERPROC) (GLuint sampler);
      |                                 ^~~~~~~~~~~~~~~~~~
include/GL/glew.h:6251:33: note: previous declaration of ‘PFNGLISSAMPLERPROC’ with type ‘PFNGLISSAMPLERPROC’ {aka ‘unsigned char (*)(unsigned int)’}
 6251 | typedef GLboolean (GLAPIENTRY * PFNGLISSAMPLERPROC) (GLuint sampler);
      |                                 ^~~~~~~~~~~~~~~~~~
include/GL/glew.h:6249:28: note: previous declaration of ‘PFNGLGETSAMPLERPARAMETERFVPROC’ with type ‘PFNGLGETSAMPLERPARAMETERFVPROC’ {aka ‘void (*)(unsigned int,  unsigned int,  float *)’}
 6249 | typedef void (GLAPIENTRY * PFNGLGETSAMPLERPARAMETERFVPROC) (GLuint sampler, GLenum pname, GLfloat* params);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:14899:28: warning: redefinition of typedef ‘PFNGLSAMPLERPARAMETERFPROC’ [-Wpedantic]
14899 | typedef void (GLAPIENTRY * PFNGLSAMPLERPARAMETERFPROC) (GLuint sampler, GLenum pname, GLfloat param);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:6254:28: note: previous declaration of ‘PFNGLSAMPLERPARAMETERFPROC’ with type ‘PFNGLSAMPLERPARAMETERFPROC’ {aka ‘void (*)(unsigned int,  unsigned int,  float)’}
 6254 | typedef void (GLAPIENTRY * PFNGLSAMPLERPARAMETERFPROC) (GLuint sampler, GLenum pname, GLfloat param);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:14897:28: warning: redefinition of typedef ‘PFNGLGETSAMPLERPARAMETERIVPROC’ [-Wpedantic]
14897 | typedef void (GLAPIENTRY * PFNGLGETSAMPLERPARAMETERIVPROC) (GLuint sampler, GLenum pname, GLint* params);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:6250:28: note: previous declaration of ‘PFNGLGETSAMPLERPARAMETERIVPROC’ with type ‘PFNGLGETSAMPLERPARAMETERIVPROC’ {aka ‘void (*)(unsigned int,  unsigned int,  int *)’}
 6250 | typedef void (GLAPIENTRY * PFNGLGETSAMPLERPARAMETERIVPROC) (GLuint sampler, GLenum pname, GLint* params);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:14900:28: warning: redefinition of typedef ‘PFNGLSAMPLERPARAMETERFVPROC’ [-Wpedantic]
14900 | typedef void (GLAPIENTRY * PFNGLSAMPLERPARAMETERFVPROC) (GLuint sampler, GLenum pname, const GLfloat* param);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:6255:28: note: previous declaration of ‘PFNGLSAMPLERPARAMETERFVPROC’ with type ‘PFNGLSAMPLERPARAMETERFVPROC’ {aka ‘void (*)(unsigned int,  unsigned int,  const float *)’}
 6255 | typedef void (GLAPIENTRY * PFNGLSAMPLERPARAMETERFVPROC) (GLuint sampler, GLenum pname, const GLfloat* params);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:14898:33: warning: redefinition of typedef ‘PFNGLISSAMPLERPROC’ [-Wpedantic]
14898 | typedef GLboolean (GLAPIENTRY * PFNGLISSAMPLERPROC) (GLuint sampler);
      |                                 ^~~~~~~~~~~~~~~~~~
include/GL/glew.h:14901:28: warning: redefinition of typedef ‘PFNGLSAMPLERPARAMETERIPROC’ [-Wpedantic]
14901 | typedef void (GLAPIENTRY * PFNGLSAMPLERPARAMETERIPROC) (GLuint sampler, GLenum pname, GLint param);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:6256:28: note: previous declaration of ‘PFNGLSAMPLERPARAMETERIPROC’ with type ‘PFNGLSAMPLERPARAMETERIPROC’ {aka ‘void (*)(unsigned int,  unsigned int,  int)’}
 6256 | typedef void (GLAPIENTRY * PFNGLSAMPLERPARAMETERIPROC) (GLuint sampler, GLenum pname, GLint param);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:14902:28: warning: redefinition of typedef ‘PFNGLSAMPLERPARAMETERIVPROC’ [-Wpedantic]
14902 | typedef void (GLAPIENTRY * PFNGLSAMPLERPARAMETERIVPROC) (GLuint sampler, GLenum pname, const GLint* param);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:6257:28: note: previous declaration of ‘PFNGLSAMPLERPARAMETERIVPROC’ with type ‘PFNGLSAMPLERPARAMETERIVPROC’ {aka ‘void (*)(unsigned int,  unsigned int,  const int *)’}
 6257 | typedef void (GLAPIENTRY * PFNGLSAMPLERPARAMETERIVPROC) (GLuint sampler, GLenum pname, const GLint* params);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:6251:33: note: previous declaration of ‘PFNGLISSAMPLERPROC’ with type ‘PFNGLISSAMPLERPROC’ {aka ‘unsigned char (*)(unsigned int)’}
 6251 | typedef GLboolean (GLAPIENTRY * PFNGLISSAMPLERPROC) (GLuint sampler);
      |                                 ^~~~~~~~~~~~~~~~~~
include/GL/glew.h:14899:28: warning: redefinition of typedef ‘PFNGLSAMPLERPARAMETERFPROC’ [-Wpedantic]
14899 | typedef void (GLAPIENTRY * PFNGLSAMPLERPARAMETERFPROC) (GLuint sampler, GLenum pname, GLfloat param);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:6254:28: note: previous declaration of ‘PFNGLSAMPLERPARAMETERFPROC’ with type ‘PFNGLSAMPLERPARAMETERFPROC’ {aka ‘void (*)(unsigned int,  unsigned int,  float)’}
 6254 | typedef void (GLAPIENTRY * PFNGLSAMPLERPARAMETERFPROC) (GLuint sampler, GLenum pname, GLfloat param);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:14900:28: warning: redefinition of typedef ‘PFNGLSAMPLERPARAMETERFVPROC’ [-Wpedantic]
14900 | typedef void (GLAPIENTRY * PFNGLSAMPLERPARAMETERFVPROC) (GLuint sampler, GLenum pname, const GLfloat* param);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:6255:28: note: previous declaration of ‘PFNGLSAMPLERPARAMETERFVPROC’ with type ‘PFNGLSAMPLERPARAMETERFVPROC’ {aka ‘void (*)(unsigned int,  unsigned int,  const float *)’}
 6255 | typedef void (GLAPIENTRY * PFNGLSAMPLERPARAMETERFVPROC) (GLuint sampler, GLenum pname, const GLfloat* params);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:14901:28: warning: redefinition of typedef ‘PFNGLSAMPLERPARAMETERIPROC’ [-Wpedantic]
14901 | typedef void (GLAPIENTRY * PFNGLSAMPLERPARAMETERIPROC) (GLuint sampler, GLenum pname, GLint param);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:6256:28: note: previous declaration of ‘PFNGLSAMPLERPARAMETERIPROC’ with type ‘PFNGLSAMPLERPARAMETERIPROC’ {aka ‘void (*)(unsigned int,  unsigned int,  int)’}
 6256 | typedef void (GLAPIENTRY * PFNGLSAMPLERPARAMETERIPROC) (GLuint sampler, GLenum pname, GLint param);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:14902:28: warning: redefinition of typedef ‘PFNGLSAMPLERPARAMETERIVPROC’ [-Wpedantic]
14902 | typedef void (GLAPIENTRY * PFNGLSAMPLERPARAMETERIVPROC) (GLuint sampler, GLenum pname, const GLint* param);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~
include/GL/glew.h:6257:28: note: previous declaration of ‘PFNGLSAMPLERPARAMETERIVPROC’ with type ‘PFNGLSAMPLERPARAMETERIVPROC’ {aka ‘void (*)(unsigned int,  unsigned int,  const int *)’}
 6257 | typedef void (GLAPIENTRY * PFNGLSAMPLERPARAMETERIVPROC) (GLuint sampler, GLenum pname, const GLint* params);
      |                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~
src/glew.c:2497:22: error: redefinition of ‘__glewBindSampler’
 2497 | PFNGLBINDSAMPLERPROC __glewBindSampler = NULL;
      |                      ^~~~~~~~~~~~~~~~~
src/glew.c:1188:22: note: previous definition of ‘__glewBindSampler’ with type ‘PFNGLBINDSAMPLERPROC’ {aka ‘void (*)(unsigned int,  unsigned int)’}
 1188 | PFNGLBINDSAMPLERPROC __glewBindSampler = NULL;
      |                      ^~~~~~~~~~~~~~~~~
src/glew.c:2498:25: error: redefinition of ‘__glewDeleteSamplers’
 2498 | PFNGLDELETESAMPLERSPROC __glewDeleteSamplers = NULL;
      |                         ^~~~~~~~~~~~~~~~~~~~
src/glew.c:1189:25: note: previous definition of ‘__glewDeleteSamplers’ with type ‘PFNGLDELETESAMPLERSPROC’ {aka ‘void (*)(int,  const unsigned int *)’}
 1189 | PFNGLDELETESAMPLERSPROC __glewDeleteSamplers = NULL;
      |                         ^~~~~~~~~~~~~~~~~~~~
src/glew.c:2499:22: error: redefinition of ‘__glewGenSamplers’
 2499 | PFNGLGENSAMPLERSPROC __glewGenSamplers = NULL;
      |                      ^~~~~~~~~~~~~~~~~
src/glew.c:1190:22: note: previous definition of ‘__glewGenSamplers’ with type ‘PFNGLGENSAMPLERSPROC’ {aka ‘void (*)(int,  unsigned int *)’}
 1190 | PFNGLGENSAMPLERSPROC __glewGenSamplers = NULL;
      |                      ^~~~~~~~~~~~~~~~~
src/glew.c:2500:32: error: redefinition of ‘__glewGetSamplerParameterfv’
 2500 | PFNGLGETSAMPLERPARAMETERFVPROC __glewGetSamplerParameterfv = NULL;
      |                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~
src/glew.c:1193:32: note: previous definition of ‘__glewGetSamplerParameterfv’ with type ‘PFNGLGETSAMPLERPARAMETERFVPROC’ {aka ‘void (*)(unsigned int,  unsigned int,  float *)’}
 1193 | PFNGLGETSAMPLERPARAMETERFVPROC __glewGetSamplerParameterfv = NULL;
      |                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~
src/glew.c:2501:32: error: redefinition of ‘__glewGetSamplerParameteriv’
 2501 | PFNGLGETSAMPLERPARAMETERIVPROC __glewGetSamplerParameteriv = NULL;
      |                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~
src/glew.c:1194:32: note: previous definition of ‘__glewGetSamplerParameteriv’ with type ‘PFNGLGETSAMPLERPARAMETERIVPROC’ {aka ‘void (*)(unsigned int,  unsigned int,  int *)’}
 1194 | PFNGLGETSAMPLERPARAMETERIVPROC __glewGetSamplerParameteriv = NULL;
      |                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~
src/glew.c:2502:20: error: redefinition of ‘__glewIsSampler’
 2502 | PFNGLISSAMPLERPROC __glewIsSampler = NULL;
      |                    ^~~~~~~~~~~~~~~
src/glew.c:1195:20: note: previous definition of ‘__glewIsSampler’ with type ‘PFNGLISSAMPLERPROC’ {aka ‘unsigned char (*)(unsigned int)’}
 1195 | PFNGLISSAMPLERPROC __glewIsSampler = NULL;
      |                    ^~~~~~~~~~~~~~~
src/glew.c:2503:28: error: redefinition of ‘__glewSamplerParameterf’
 2503 | PFNGLSAMPLERPARAMETERFPROC __glewSamplerParameterf = NULL;
      |                            ^~~~~~~~~~~~~~~~~~~~~~~
src/glew.c:1198:28: note: previous definition of ‘__glewSamplerParameterf’ with type ‘PFNGLSAMPLERPARAMETERFPROC’ {aka ‘void (*)(unsigned int,  unsigned int,  float)’}
 1198 | PFNGLSAMPLERPARAMETERFPROC __glewSamplerParameterf = NULL;
      |                            ^~~~~~~~~~~~~~~~~~~~~~~
src/glew.c:2504:29: error: redefinition of ‘__glewSamplerParameterfv’
 2504 | PFNGLSAMPLERPARAMETERFVPROC __glewSamplerParameterfv = NULL;
      |                             ^~~~~~~~~~~~~~~~~~~~~~~~
src/glew.c:1199:29: note: previous definition of ‘__glewSamplerParameterfv’ with type ‘PFNGLSAMPLERPARAMETERFVPROC’ {aka ‘void (*)(unsigned int,  unsigned int,  const float *)’}
 1199 | PFNGLSAMPLERPARAMETERFVPROC __glewSamplerParameterfv = NULL;
      |                             ^~~~~~~~~~~~~~~~~~~~~~~~
src/glew.c:2505:28: error: redefinition of ‘__glewSamplerParameteri’
 2505 | PFNGLSAMPLERPARAMETERIPROC __glewSamplerParameteri = NULL;
      |                            ^~~~~~~~~~~~~~~~~~~~~~~
src/glew.c:1200:28: note: previous definition of ‘__glewSamplerParameteri’ with type ‘PFNGLSAMPLERPARAMETERIPROC’ {aka ‘void (*)(unsigned int,  unsigned int,  int)’}
 1200 | PFNGLSAMPLERPARAMETERIPROC __glewSamplerParameteri = NULL;
      |                            ^~~~~~~~~~~~~~~~~~~~~~~
src/glew.c:2506:29: error: redefinition of ‘__glewSamplerParameteriv’
 2506 | PFNGLSAMPLERPARAMETERIVPROC __glewSamplerParameteriv = NULL;
      |                             ^~~~~~~~~~~~~~~~~~~~~~~~
src/glew.c:1201:29: note: previous definition of ‘__glewSamplerParameteriv’ with type ‘PFNGLSAMPLERPARAMETERIVPROC’ {aka ‘void (*)(unsigned int,  unsigned int,  const int *)’}
 1201 | PFNGLSAMPLERPARAMETERIVPROC __glewSamplerParameteriv = NULL;
      |                             ^~~~~~~~~~~~~~~~~~~~~~~~
src/glew.c:2497:22: error: redefinition of ‘__glewBindSampler’
 2497 | PFNGLBINDSAMPLERPROC __glewBindSampler = NULL;
      |                      ^~~~~~~~~~~~~~~~~
src/glew.c:1188:22: note: previous definition of ‘__glewBindSampler’ with type ‘PFNGLBINDSAMPLERPROC’ {aka ‘void (*)(unsigned int,  unsigned int)’}
 1188 | PFNGLBINDSAMPLERPROC __glewBindSampler = NULL;
      |                      ^~~~~~~~~~~~~~~~~
src/glew.c:2498:25: error: redefinition of ‘__glewDeleteSamplers’
 2498 | PFNGLDELETESAMPLERSPROC __glewDeleteSamplers = NULL;
      |                         ^~~~~~~~~~~~~~~~~~~~
src/glew.c:1189:25: note: previous definition of ‘__glewDeleteSamplers’ with type ‘PFNGLDELETESAMPLERSPROC’ {aka ‘void (*)(int,  const unsigned int *)’}
 1189 | PFNGLDELETESAMPLERSPROC __glewDeleteSamplers = NULL;
      |                         ^~~~~~~~~~~~~~~~~~~~
src/glew.c:2499:22: error: redefinition of ‘__glewGenSamplers’
 2499 | PFNGLGENSAMPLERSPROC __glewGenSamplers = NULL;
      |                      ^~~~~~~~~~~~~~~~~
src/glew.c:1190:22: note: previous definition of ‘__glewGenSamplers’ with type ‘PFNGLGENSAMPLERSPROC’ {aka ‘void (*)(int,  unsigned int *)’}
 1190 | PFNGLGENSAMPLERSPROC __glewGenSamplers = NULL;
      |                      ^~~~~~~~~~~~~~~~~
src/glew.c:2500:32: error: redefinition of ‘__glewGetSamplerParameterfv’
 2500 | PFNGLGETSAMPLERPARAMETERFVPROC __glewGetSamplerParameterfv = NULL;
      |                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~
src/glew.c:1193:32: note: previous definition of ‘__glewGetSamplerParameterfv’ with type ‘PFNGLGETSAMPLERPARAMETERFVPROC’ {aka ‘void (*)(unsigned int,  unsigned int,  float *)’}
 1193 | PFNGLGETSAMPLERPARAMETERFVPROC __glewGetSamplerParameterfv = NULL;
      |                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~
src/glew.c:2501:32: error: redefinition of ‘__glewGetSamplerParameteriv’
 2501 | PFNGLGETSAMPLERPARAMETERIVPROC __glewGetSamplerParameteriv = NULL;
      |                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~
src/glew.c:1194:32: note: previous definition of ‘__glewGetSamplerParameteriv’ with type ‘PFNGLGETSAMPLERPARAMETERIVPROC’ {aka ‘void (*)(unsigned int,  unsigned int,  int *)’}
 1194 | PFNGLGETSAMPLERPARAMETERIVPROC __glewGetSamplerParameteriv = NULL;
      |                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~
src/glew.c:2502:20: error: redefinition of ‘__glewIsSampler’
 2502 | PFNGLISSAMPLERPROC __glewIsSampler = NULL;
      |                    ^~~~~~~~~~~~~~~
src/glew.c:1195:20: note: previous definition of ‘__glewIsSampler’ with type ‘PFNGLISSAMPLERPROC’ {aka ‘unsigned char (*)(unsigned int)’}
 1195 | PFNGLISSAMPLERPROC __glewIsSampler = NULL;
      |                    ^~~~~~~~~~~~~~~
src/glew.c:2503:28: error: redefinition of ‘__glewSamplerParameterf’
 2503 | PFNGLSAMPLERPARAMETERFPROC __glewSamplerParameterf = NULL;
      |                            ^~~~~~~~~~~~~~~~~~~~~~~
src/glew.c:1198:28: note: previous definition of ‘__glewSamplerParameterf’ with type ‘PFNGLSAMPLERPARAMETERFPROC’ {aka ‘void (*)(unsigned int,  unsigned int,  float)’}
 1198 | PFNGLSAMPLERPARAMETERFPROC __glewSamplerParameterf = NULL;
      |                            ^~~~~~~~~~~~~~~~~~~~~~~
src/glew.c:2504:29: error: redefinition of ‘__glewSamplerParameterfv’
 2504 | PFNGLSAMPLERPARAMETERFVPROC __glewSamplerParameterfv = NULL;
      |                             ^~~~~~~~~~~~~~~~~~~~~~~~
src/glew.c:1199:29: note: previous definition of ‘__glewSamplerParameterfv’ with type ‘PFNGLSAMPLERPARAMETERFVPROC’ {aka ‘void (*)(unsigned int,  unsigned int,  const float *)’}
 1199 | PFNGLSAMPLERPARAMETERFVPROC __glewSamplerParameterfv = NULL;
      |                             ^~~~~~~~~~~~~~~~~~~~~~~~
src/glew.c:2505:28: error: redefinition of ‘__glewSamplerParameteri’
 2505 | PFNGLSAMPLERPARAMETERIPROC __glewSamplerParameteri = NULL;
      |                            ^~~~~~~~~~~~~~~~~~~~~~~
src/glew.c:1200:28: note: previous definition of ‘__glewSamplerParameteri’ with type ‘PFNGLSAMPLERPARAMETERIPROC’ {aka ‘void (*)(unsigned int,  unsigned int,  int)’}
 1200 | PFNGLSAMPLERPARAMETERIPROC __glewSamplerParameteri = NULL;
      |                            ^~~~~~~~~~~~~~~~~~~~~~~
src/glew.c:2506:29: error: redefinition of ‘__glewSamplerParameteriv’
 2506 | PFNGLSAMPLERPARAMETERIVPROC __glewSamplerParameteriv = NULL;
      |                             ^~~~~~~~~~~~~~~~~~~~~~~~
src/glew.c:1201:29: note: previous definition of ‘__glewSamplerParameteriv’ with type ‘PFNGLSAMPLERPARAMETERIVPROC’ {aka ‘void (*)(unsigned int,  unsigned int,  const int *)’}
 1201 | PFNGLSAMPLERPARAMETERIVPROC __glewSamplerParameteriv = NULL;
      |                             ^~~~~~~~~~~~~~~~~~~~~~~~
make: *** [Makefile:134: tmp/linux-egl-glvnd/default/static/glew.o] Error 1
make: *** Waiting for unfinished jobs....
make: *** [Makefile:138: tmp/linux-egl-glvnd/default/shared/glew.o] Error 1
```

Any idea on what's going on here, please?

Thanks!

**Comment by vanfanel** [comment]
No matter how much back in time I go, it doesn't build anymore.
I suspect some upstream problem with https://github.com/KhronosGroup/OpenGL-Registry.git, could that be the case?

**Comment by vanfanel** [comment]
This is the output of `make extensions`, many `no keys in table!` errors apparently, is that expected?

[make_extensions_output.txt](https://github.com/nigels-com/glew/files/13311308/make_extensions_output.txt)

**Comment by geffrak** [comment]
I'm having the same issue while trying to build as a dependency of our app. make 
from auto doesnt pull a fixed version of https://github.com/KhronosGroup/OpenGL-Registry.git. The new commit https://github.com/KhronosGroup/OpenGL-Registry/commit/79d456c3cadd70d5f79c1d045610328d005c466e introduced new symbols that are now duplicated in glew.c

**Comment by nigels-com** [comment]
Fixed via #401 .  Can you confirm on your side?
Thanks for raising the alarm!

**Comment by vanfanel** [comment]
@nigels-com Builds perfectly now, it's solved via that commit! Thanks! Always a pleasure to help :)
